test/common/odp_cunit_common.c

```diff
@@ -733,6 +733,11 @@ int odp_cunit_ret(int val)
 	return allow_skip_result ? 0 : val;
 }
 
+int odp_cunit_ci(void)
```

Comment:
Since "skip result" is miss leading here, a comment about purpose of the function could be added. E.g. "Checks if test application is running in CI"

Reply:
I changed the name of the variable in v5.